drivers/lhal/src/bflb_gpio.c

```diff
@@ -86,6 +104,40 @@ void bflb_gpio_init(struct bflb_device_s *dev, uint8_t pin, uint32_t cfgset)
         cfg |= (1 << (is_odd * 16 + 15));
     }
 #endif
+    /* always on pads IE control (in HBN) */
```

Comment:
lhal gpio is for common driver, so these give up, please use soc api GLB_GPIO_Init for these gpio.

Reply:
No, it is not common, because it has many many ifdef's in it for various MCU models and kinds. So I'm sure, that bflb_gpio_init() should match GLB_GPIO_Init() in case of GLB_GPIO_USE_PSRAM__IO handling.